ui/src/dashboards/selectors/index.test.ts

```diff
@@ -18,34 +24,99 @@ const untypedGetTimeRangeByDashboardID = getTimeRange as (a: {
   currentDashboard: CurrentDashboardState
 }) => TimeRange
 
+const untypedGetTimeRangeWithTimeZone = getTimeRangeWithTimezone as (a: {
+  ranges: RangeState
+  currentDashboard: CurrentDashboardState
+  app: AppPresentationState
+}) => TimeRange
+
 describe('Dashboards.Selector', () => {
-  const dashboardIDs = ['04c6f3976f4b8001', '04c6f3976f4b8000']
+  const dashboardIDs = [
+    '04c6f3976f4b8001',
+    '04c6f3976f4b8000',
+    '04c6f3976f4b8002',
+  ]
+  const lower = '2020-05-05T10:00:00-07:00'
+  const upper = '2020-05-05T11:00:00-07:00'
+  const customTimeRange = {
+    lower: moment(lower).toISOString(),
+    upper: moment(upper).toISOString(),
+    type: 'custom',
```

Comment:
by setting these values to `ISOString` more accurately reflects how the values are represented when passing them to the function